src/pycountry/db.py

```diff
@@ -155,7 +155,7 @@ def __len__(self) -> int:
 
     @lazy_load
     def get(
-        self, *, default: Optional[Any] = None, **kw: Optional[str]
+        self, *, default: Optional[Any] = [], **kw: Optional[str]
```

Comment:
It's a common pitfall to use a mutable collection as a default argument. [This is a good article on it.](https://www.pullrequest.com/blog/python-pitfalls-the-perils-of-using-lists-and-dicts-as-default-arguments/)

Instead you should check if `default` is `None`, and if it is, set its value to a new empty list.

Example:

```
def append_item(item, item_list=None):
    if item_list is None:
        item_list = []  # Create a new list if no list is provided
    item_list.append(item)
    return item_list
```